tensorflow/core/util/gpu_kernel_helper.h

```diff
@@ -46,10 +46,12 @@ limitations under the License.
 #if GOOGLE_CUDA
 #define gpuSuccess cudaSuccess
 using gpuStream_t = cudaStream_t;
+#define GPU_GET_ERROR_STRING(error) cudaGetErrorString(error)
```

Comment:
@chsigg and @jerryyin had some discussion about this in the earlier PR IIRC.

Reply:
Right... I used to add gpuGetErrorString in this [PR](https://github.com/tensorflow/tensorflow/pull/28568/files). Not sure why it is gone in this file now...

Reply:
interesting....it seems some of the changes in that PR got dropped during the merge :(

let me revise this PR to use the version of this routine already approved in that PR.

Reply:
Thanks! Leave the `GetGpuStream()` to me. I'll make it a standalone PR together with updates from callsites.

Reply:
it seems that the merge for PR #28568 did happen correctly (i.e. no changes were dropped). Instead it was the subsequent merge for PR @28565 that somehow dropped some of the changes.

Reply:
pushed out the version from PR #28568